.github/workflows/docker-compose-enterprise.yml

```diff
@@ -48,9 +52,29 @@ jobs:
         && github.event.pull_request.head.user.login == 'Alfresco'
       )
     steps:
-      - uses: >-
-          Alfresco/alfresco-build-tools/.github/actions/dbp-charts/verify-compose@v5.12.0
+      - name: Verify docker-compose
+        uses: >-
+          Alfresco/alfresco-build-tools/.github/actions/dbp-charts/verify-compose@v5.13.0
+        timeout-minutes: 10
         with:
           compose_file_path: docker-compose/${{ matrix.compose_file }}
           quay_username: ${{ secrets.QUAY_USERNAME }}
           quay_password: ${{ secrets.QUAY_PASSWORD }}
+
+      - name: Show docker ps
+        if: always()
+        working-directory: docker-compose
+        run: |
+          docker-compose -f ${{ matrix.compose_file }} ps
+
+      - name: Show alfresco logs
+        if: always()
+        working-directory: docker-compose
+        run: |
+          docker-compose -f ${{ matrix.compose_file }} logs alfresco
+
+      - name: Show solr logs
+        if: always()
+        working-directory: docker-compose
+        run: |
+          docker-compose -f ${{ matrix.compose_file }} logs solr6
```

Comment:
Why specificly those?
We could get all logs and upload a artifacts instead?

Reply:
indeed but I guess at this point that the issue has been solved are not really worth keeping it

I would like to improve the dbp-charts/verify-compose first which ATM is just a monolith bash script, could be a fit for compose improvements epic